bundles/org.eclipse.equinox.p2.core/src/org/eclipse/equinox/p2/core/IProvisioningAgent.java

```diff
@@ -165,4 +165,32 @@ default boolean getBooleanProperty(String key, boolean defaultValue) {
 		return Boolean.parseBoolean(getProperty(key, Boolean.toString(defaultValue)));
 	}
 
+	/**
+	 * Returns an agent bound property as an int, the default implementation
+	 * delegates to {@link IProvisioningAgent#getIntProperty(String, int)} with
+	 * <code>0</code> as the default.
+	 *
+	 * @since 2.13
+	 */
+	default int getIntProperty(String key) {
+		return getIntProperty(key, 0);
+	}
```

Comment:
This does no harm but it's arguably overkill because what are the odds that we will need an integer property that default to 0.

Reply:
So I guess I should remove it? I added this to imitate boolean properties.

Reply:
Good point about getBoolean.  As such, I think it's fine and good to leave it.  Best it all be parallel.